### PR TITLE
Task 4-6 Completed

### PR DESCRIPTION
**Pull request by MenacingManatee**

### CODE REVIEW

**Comment by UsmanAJabbar** [comment]
# HIGHLIGHTS
- Live version of the flask server could be accessed at [pro.geeksultd.com:5000](url)
- Added code for task 4-6.
- All files have been PEP8 checked.
- .gitignore added to skip over pycache folders.

File: api/v1/app.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python3
+"""DOCSTRING"""
+from flask import Flask, jsonify
+from models import storage
+from api.v1.views import app_views
+
+
+app = Flask(__name__)
```
Comment (MenacingManatee) [inline api/v1/app.py:8]: Just wanna point out, I think this work was supposed to be on a separate branch. Storage_get_count is gonna be manual reviewed, and I think it was only supposed to have the get() and count() methods on it

File: api/v1/app.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+@app.teardown_appcontext
+def teardown_fs_db(exception):
+    """closes the storage on teardown"""
+    storage.close()
+
+@app.errorhandler(404)
+def api_error(self):
+    """Returns an error page when pinged"""
+    return jsonify({ "error": "Not Found" }), 404
```
Comment (MenacingManatee) [inline api/v1/app.py:20]: minor nitpick, but is the spaces on the inside of the curly braces needed? It looks odd to me

File: api/v1/views/index.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python3
+"""DUMMY DOX"""
```
Comment (MenacingManatee) [inline api/v1/views/index.py:2]: Dummy dox? lmao

**Pull request merged**